.github/workflows/build-and-test.yml

```diff
@@ -165,10 +165,6 @@ jobs:
         run: |
           make -j2 goporto
           git diff --exit-code || (echo 'Porto links are out of date, please run "make goporto" and commit the changes in this PR.' && exit 1)
-      - name: crosslink
-        run: |
-          make crosslink
-          git diff --exit-code || (echo 'Replace statements are out of date, please run "make crosslink" and commit the changes in this PR.' && exit 1)
```

Comment:
Why is this check removed?

Reply:
I removed all of crosslink, since it used the root module as its reference and we’re removing most of the root go.mod.

crosslink and genotelcontribcol would conflict each other.

Reply:
`crosslink` is a very helpful tool to keep the "replace" statements tidy. I'm not sure it can be replaced with `genotelcontribcol`. Is there a to make `crosslink` work with a non-root as its reference?

Reply:
I don’t think so? It doesn’t look possible sorry.

Reply:
Hi! I'm one of the primary authors of `crosslink`. This change could be made in crosslink if we want to support this use case. Where a root `go.mod` file does not exist. It would require some code changes within crosslink. If this is desired I we should open an issue in go-build-tools asking for this feature and describing its use case. I think it would be a low/med effort to update crosslink to do this.

Reply:
we can look into it but I think intentionally slowing down before we bug you is the right move. This PR is a spike and I have had success breaking it into smaller chunks so far. Thanks for reaching out @bryan-aguilar !